modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -180,22 +217,30 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
      * @param null $test_name   the value of test name
      * @param null $candID      the value of candidate ID
      * @param null $visit_label the value of visit label
+     * @param null $site        the value of site
+     * @param null $project     the value of project
      *
      * @return   Array containing a list of feedbacks for
      * the appropriate feedback level.
      * @throws   DatabaseException
      * @internal param unknown_type $field_name
      */
-    function getFeedbacks($test_name = null, $candID=null, $visit_label=null)
-    {
+    function getFeedbacks($test_name = null,
+        $candID=null,
+        $visit_label=null,
+        $site=null,
+        $project=null
+    ) {
 
         $DB      =& Database::singleton();
         $qparams = array();
 
         $select = "SELECT fb.FeedbackID, fb.CandID, fb.SessionID,
                       fb.CommentID, fb.Feedback_level, f.Test_name, fb.FieldName,
-                      tn.Full_name, s.Visit_label FROM feedback_bvl_thread AS fb
+                      tn.Full_name, s.Visit_label, s.PSCID 
```

Comment:
The module is broken for me on this line. `s.PSCID` doesn't seem to work.
<img width="878" alt="screen shot 2017-05-04 at 11 52 12 am" src="https://cloud.githubusercontent.com/assets/4022790/25712695/52677ab0-30c0-11e7-8800-20ed3b3aee29.png">
<img width="882" alt="screen shot 2017-05-04 at 11 52 54 am" src="https://cloud.githubusercontent.com/assets/4022790/25712700/53f1a2de-30c0-11e7-9a7e-7a4c85b90ad8.png">